test/js/ui/map.test.js

```diff
@@ -103,7 +98,7 @@ test('Map', function(t) {
             style.fire('source.error');
             style.fire('source.change');
             style.fire('tile.add');
-            style.fire('tile.load');
+            // style.fire('tile.load');
```

Comment:
Commenting out code is bad practice. 

Can we delete this line and do one of

 - accept that this event isn't fired 
 - open an issue about improving our even testing 
 - improve our event testing in this PR